src/test/scala/edu/ie3/simona/model/participant/PvModelSpec.scala

```diff
@@ -777,56 +776,57 @@ class PvModelSpec extends UnitSpec with GivenWhenThen with DefaultTestData {
       )
 
       forAll(testCases) {
-        (thetaGDeg, thetaZDeg, gammaEDeg, airMass, I0, eDifSSol) =>
+        (thetaGDeg, thetaZDeg, gammaEDeg, airMass, g0, gDifSSol) =>
           // Reference Duffie 4th ed., p.95
           Given("using the input data")
-          // Beam Radiation on horizontal surface
-          val eBeamH =
-            megaJoule2WattHours(0.244) // 0.244 MJ/m^2 = 67.777778 Wh/m^2
-          // Diffuse Radiation on a horizontal surface
-          val eDifH =
-            megaJoule2WattHours(0.796) // 0.796 MJ/m^2 = 221.111111 Wh/m^2
-
-          When("the diffuse radiation is calculated")
-          val eDifSCalc = pvModel.calcDiffuseRadiationOnSlopedSurfacePerez(
-            WattHoursPerSquareMeter(eDifH),
-            WattHoursPerSquareMeter(eBeamH),
+          // Beam irradiance on horizontal surface given a radiation for one hour
+          // 0.244 MJ/m^2 = 67.777778 Wh/m^2
+          val gBeamH = megaJoule2WattHours(0.244)
```

Comment:
I think, we should add another method, since the name of this method does not fit (`2WattHours`).

Reply:
I agree the situation is not quite ideal here. I added some commentary and replaced the conversion function with squants functionality.